mmv1/third_party/terraform/services/compute/resource_compute_region_instance_group_manager_test.go.tmpl

```diff
@@ -329,6 +338,15 @@ func TestAccRegionInstanceGroupManager_distributionPolicy(t *testing.T) {
 				ImportStateVerify:       true,
 				ImportStateVerifyIgnore: []string{"status"},
 			},
+			{
+				Config: testAccRegionInstanceGroupManager_distributionPolicyEmpty(template, igm),
```

Comment:
I feel stupid that I don't know this but does this test tests the updates or only creation?

Reply:
I think we should generally follow the pattern (in tests):
1. empty/unspecified field
2. some value
3. other value
4. empty/unspecified (to check if we're not overwriting/correctly deleting)
Generally we're applying all the configurations specified in testAccRegionInstanceGroupManager_distributionPolicy (including the one you commented on), and checking if nothing seems to be failing.